### DISCUSSION TITLE
Where do i download kext file?

### DISCUSSION THREAD

**Discussion by Ajay-CR** [ChefKissInc/NootedRed#170] [General]
cant find it. I am using 2200G. I successfully installed Ventura. Any suggestion help would be great!

**Comment by truongvantuan** [comment]
> cant find it. I am using 2200G. I successfully installed Ventura. Any suggestion help would be great!

Action tab

**Comment by truongvantuan** [comment]
> cant find it. I am using 2200G. I successfully installed Ventura. Any suggestion help would be great!

https://github.com/ChefKissInc/NootedRed/actions/runs/6157959609

**Reply by Ajay-CR** [reply]
Thanks, @truongvantuan
